features/fixtures/ast.json

```diff
@@ -15,18 +15,20 @@
       "description": "<resource group description>\n\n",
       "resources": [
         {
+          "element": "resource",
           "name": "<resource name>",
           "description": "<resource description>\n\n",
           "uriTemplate": "/resource/{parameter}",
           "model": {
             "name": "<resource name>",
-            "description": "<resource model description>",
+            "description": "<resource model description>\n",
```

Comment:
It was 

```
+ Model
    <resource model description>
```

The description would be considered a part of the `Model` markdown node. I just added a new line in between so that description is a new paragraph markdown node.